lib/foodcritic/command_line.rb

```diff
@@ -76,7 +77,8 @@ def initialize(args)
           @options[:version] = true
         end
         opts.on('-X', '--exclude PATH',
-                'Exclude path(s) from being linted.') do |e|
+                'Exclude path(s) from being linted. Default test/**/*,'\
+                'spec/**/*,features/**/*') do |e|
```

Comment:
Given how this works, it looks like it might not be possible to remove the default? We should probably add something for that.